pkgs/applications/virtualization/docker/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     patchShebangs .
     export AUTO_GOPATH=1
-    export DOCKER_GITCOMMIT="a34a1d59"
```

Comment:
No it isn't related to the the fish completions. I just noticed that it was out of sync with the actual version we use and updated it. Presumably it needs to be changed again shortly, when someone updates to 11.1.
The value shows up in `docker -v`. I imagine some people expect it there. I don't really see a problem with keeping it in sync whenever docker is updated, it seems manageable.